pkg/object/meshcontroller/spec/spec.go

```diff
@@ -175,6 +181,22 @@ type (
 		URLs                  []*urlrule.URLRule              `yaml:"urls" jsonschema:"required"`
 	}
 
+	// ServiceCanary is the service canary entry.
+	ServiceCanary struct {
+		Name string `yaml:"name" jsonschema:"required"`
+		// Priority must be [1, 9], the default is 5 if user does not set it.
+		// The smaller number get higher priority.
+		// The order is sorted by name alphabetically in the same priority.
```

Comment:
why limit priority to [1, 9], is the range large enough?

Reply:
Enough for our scenarios. We leverage the OS scheduling system plus explicit two-layer sorting. Infinite priority with integer causes a bad user experience.